src/lib/load-js.js

```diff
@@ -0,0 +1,6 @@
+/* @flow */
+
+export const loadJavascript = (url : string) => {
```

Comment:
Would be good if we could make this non-minified and just have the compiler take care of this. Difficult to see exactly what this function is doing otherwise.

Reply:
done